### ISSUE TITLE
Error: Cannot query field "allMarkdownRemark" on type "Query".

### ISSUE THREAD

**Issue by hsavit1** [kata-ai/kata-platform-docs#16]
Ran into this error after running `yarn` and `yarn dev`

`Error: Cannot query field "allMarkdownRemark" on type "Query".`

```
⠁ [ { GraphQLError: Cannot query field "allMarkdownRemark" on type "Query".
      at Object.Field (/Users/x/xx/Documents/kata-platform-docs/node_modules/graphql/validation/rules/FieldsOnCorrectType.js:65:31)
.
.
.
.
    message: 'Cannot query field "allMarkdownRemark" on type "Query".',
    locations: [ [Object] ],
    path: undefined } ]
```

**Comment by resir014** [comment]
Hi, could you make sure you're running on Node.js 8+ or above? If the problem still occurs, please try removing both the `node_modules` and `.gatsby` directory (if it exists) and rerun `yarn` and `yarn dev`.

**Comment by necheporenko** [comment]
Using plugin gatsby-source-filesystem add also path to your folder with .md files.

**Comment by resir014** [comment]
Should be fixed with #48.

**Comment by fraserisland** [comment]
Have https://www.gatsbyjs.org/packages/gatsby-transformer-remark/ installed?
